src/main/perl/Path.pm

```diff
@@ -346,6 +352,21 @@ sub any_exists
     return $path && (-e $path || -l $path);
 }
 
+=item is_symlink
+
+Test if C<path> is a symlink.
+
+Returns true as long as C<path> is a symlink, even though the 
+symlink target doesn't exist.
+
+=cut
+
+sub is_symlink
```

Comment:
out of curiosity, why is there no `is_hardlink`?  maybe not `is_hardlink`, but rather `has_hardlink`.
i'm not saying we should have one, i'm just wondering.